sdk/storage/azure-storage-blob/tests/avro/test_avro_async.py

```diff
@@ -84,9 +84,8 @@ async def _test_reader(self):
             correct,
             len(CODECS_TO_VALIDATE) * len(SCHEMAS_TO_VALIDATE))
 
-    def test_reader(self):
-        loop = asyncio.get_event_loop()
-        loop.run_until_complete(self._test_reader())
+    async def test_reader(self):
+        await self._test_reader()
```

Comment:
I would say get rid of these methods all together and just get rid of `_` in `_test_reader` to make that the test that is automatically run. No need for this helper method now :)

Reply:
Ah gotcha. I remember us chatting about this offline and was slightly confused but this clears it up 😄